en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -300,7 +300,6 @@ table may be of use:
 
 | Silverstripe CMS Version | PHP Version | More information |
 | ------------------------ | ----------- | ---------------- |
-| 5.0 +                    | 8.1 - 8.2   | Unreleased       |
```

Comment:
This would get outdated quickly - and we don't want to update CMS 4 docs when we do stuff in CMS 5.